Enhancement: introduce zip parser to add documents from within zip files (#5570) (#10274)
paperless-ngx/paperless-ngx: docs/faq.md

-   With the optional Tika integration enabled (see [Tika configuration](https://docs.paperless-ngx.com/configuration#tika)),
    Paperless also supports various Office documents (.docx, .doc, odt,
    .ppt, .pptx, .odp, .xls, .xlsx, .ods).

> -   Zip files are now supported as well. All file types mentioned above

Reviewer: ```suggestion
-   Zip files are supported as well. All file types mentioned above
```

>     found in the zip will be extracted and added to paperless.

Reviewer: ```suggestion
    found in the zip will be extracted and queued for consumption.
```
